app/lib/utils.dart

```diff
@@ -51,3 +51,56 @@ String canonicalizeVersion(String version) {
   }
   return canonicalVersion.toString();
 }
+
+RegExp _identifierExpr = new RegExp(r'^[a-zA-Z0-9_]*$');
+RegExp _startsWithLetterOrUnderscore = new RegExp(r'^[a-zA-Z_]');
+const _reservedWords = const [
```

Comment:
Consider also typing it `const <String> [ ... ]`. Now that we are moving towards the strong mode type system, it's probably better.